### PR TITLE
Add check for minimum mollie version

### PR DESCRIPTION
**Pull request by Dinamiko**
ECUR-55

### CODE REVIEW

File: src/EnvironmentChecker.php [hunk 1]
```diff
@@ -4,17 +4,33 @@
...
+    {
+        return is_plugin_active('mollie-payments-for-woocommerce/mollie-payments-for-woocommerce.php');
+    }
+
+    /**
+     * @return bool Whether current Mollie version is allowed
+     */
+    public function isMollieMinimalVersion(): bool
+    {
+        $molliePluginData = get_plugin_data(M4W_FILE);
```
Comment (strangerkir) [inline src/EnvironmentChecker.php:27]: Probably it worth it to check if that constant exists? I guess it may cause problems in case of uncareful usage.

**Review by strangerkir — CHANGES_REQUESTED** [review]
Looks good to me, just one detail I have doubts about.

File: src/EnvironmentChecker.php (hunk 1 above)
Comment (Dinamiko) [reply src/EnvironmentChecker.php:31]: Yup, good catch. thanks :)

**Review by strangerkir — APPROVED** [review]
Looks good to me.

**Pull request merged**